okhttp/src/main/java/okhttp3/internal/concurrent/Task.kt

```diff
@@ -56,8 +56,6 @@ abstract class Task(
   /** Undefined unless this is in [TaskQueue.futureTasks]. */
   internal var nextExecuteNanoTime = -1L
 
-  internal var runRunnable: Runnable? = null
-
   /** Returns the delay in nanoseconds until the next execution, or -1L to not reschedule. */
```

Comment:
nit: Negative values < -1 seem to be effectively a priority, is that ok? Or should be an error?

Not a big concern given this is internal.

Reply:
Intended contract is for implementations to never return such values! If ever we wanted to make this more broadly usable we’d need to enforce that requirement.